Documentation/Dockerfile

```diff
@@ -4,6 +4,8 @@ LABEL maintainer="maintainer@cilium.io"
 
 RUN apk add --no-cache --virtual --update \
     aspell-en \
+    nodejs \
+    npm \
```

Comment:
Are these dependencies required for the new theme?

Reply:
yes, it's built time dependency

Reply:
@genbit would this break `make live-preview`?

Reply:
not sure if this is the issue, but RTD build fails:

https://readthedocs.org/api/v2/build/11177303.txt

Reply:
hmm, i'm not sure how it ever worked :) @genbit aren't we supposed to be using python 3?

Reply:
I suspect that we make use of `six` to allow the code to run on both python2 and python3. RTD still runs python2 according to the log (not sure if there's a way to customize this, maybe there is)

Reply:
I found an option in the RTD options to switch to Python3, will retry.

Reply:
The build worked with Python3, but the theme doesn't seem to be the same as the netlify example: https://joestringercilium.readthedocs.io/en/review-docs-redesign/

Reply:
> @genbit would this break make live-preview?

potentially yes, it works on netlify, but locally you would need nodejs to build theme static files.

Reply:
Here's what happens when you do live-preview:

```
$ make -C Documentation live-preview

+--------- manually triggered build ---------------------------------------------

Theme error:
no theme named 'sphinx_rtd_theme_cilium' found (missing theme.conf?)
+--------------------------------------------------------------------------------
```

Then it renders with the default sphinx theme.

Reply:
This might fix live-preview: https://github.com/cilium/cilium/pull/11940